# Msg defining a 3D bounding box
geometry_msgs/Pose center  # 3D position and orientation of the bounding box center
geometry_msgs/Vector3 size # The size of the bounding box, in meters
